tests/lib/managers/webhooks-test.js

```diff
@@ -153,6 +153,26 @@ describe('Webhooks', function() {
 
 	});
 
+	describe('setSignatureKeys()', function() {
+		const PRIMARY_SIGNATURE_KEY = 'SamplePrimaryKey';
+		const SECONDARY_SIGNATURE_KEY = 'SampleSecondaryKey';
+
+		it('should validate the primary signature key is set', function() {
```

Comment:
This test name isn't very descriptive; maybe something like "should set the primary and secondary keys when both are passed"?